src/thumbnail_data.js

```diff
@@ -369,8 +366,8 @@ class thumbnail_data
             else
                 throw "Unrecognized source: " + source;
 
-            // Different APIs return different thumbnail URLs.
-            remapped_thumb_info.url = remapped_thumb_info.url.replace(/\/240x240\//, "/540x540_70/");
+            // Switch the URL from the low-res thumbnail to a higher-res one.
```

Comment:
This was fixed a couple months ago, I think your tree is really far behind...

Reply:
1.5 month to be precise. I didn't see it until this year.